src/main/java/com/sinthoras/visualprospecting/mixinplugin/TargetedMod.java

```diff
@@ -38,7 +39,7 @@ public boolean isMatchingJar(Path path) {
         final String nameLowerCase = Files.getNameWithoutExtension(pathString).toLowerCase();
         final String fileExtension = Files.getFileExtension(pathString);
         
-        return nameLowerCase.startsWith(jarNamePrefixLowercase) && "jar".equals(fileExtension);
+        return nameLowerCase.startsWith(jarNamePrefixLowercase) && ("jar".equals(fileExtension) || "litemod".equals(fileExtension));
```

Comment:
Does this change apply to all litemod loaded mods? Will this check handle all possible mods well? Then I'd propose it to ExampleMod as well!

Reply:
As far as I know that is the case, yes